Development

keshav-c opened the pull request:
Awaiting your review. You can see the live version here: https://keshav-c.github.io/nytimes/

keshav-c committed:
Add directory structure

keshav-c committed:
Page header in progress

mosesogwo committed:
started article section

mosesogwo committed:
more content in article

mosesogwo committed:
add all content for article

keshav-c committed:
Progress on positioning

Add aside. Divide article into sections. Adjust figure positions.
Adjust caption positions. Add external fonts for figure captions.

mosesogwo committed:
chart-texts added, positioning in progress

keshav-c committed:
Start recomendation section etc

Add iframe for external content. Style show comments button. Start
adding recommendation section.

mosesogwo committed:
section-reco in place

mosesogwo committed:
section-reco in progress

keshav-c committed:
Finish recommendations, footer and header

Finish aligning and adding the recommendation section, navigation
section and the page footer. centre content in page header.

keshav-c committed:
Indent markup, cleanup comments in css

keshav-c committed:
Order declarations alphabetically in CSS

keshav-c committed:
Combine redundant selectors add a tags to recommend section

keshav-c committed:
Add Linter recommended changes

Rulesets in alphabetical order. No margin on inline blocks. No
qualifying h1.

keshav-c committed:
Replace h4 in CSS with aside-heading class

keshav-c committed:
Remove individual h4 styles from CSS

mcrd25 commented on index.html:
```diff
@@ -0,0 +1,402 @@
+                    <aside class="editors-picks">
+                        <h4>Editor's Picks</h4>
+                        <div>
+                            <a href="article1.html"></a>
```
Are the `img` and `p` tags supposed to be wrapped inside this `a` tag? if so you have accidentally closed it. 😅 
Happens. Please fix 👍 .

mcrd25 commented on index.html:
```diff
@@ -0,0 +1,402 @@
+                </div>
+                <div>
+                    <a href="#">
+                        <figure>
```
I like that you have used `figure` tag in this project. Good job exploring different HTML tags 👍

mcrd25 commented on index.html:
```diff
@@ -0,0 +1,402 @@
+                </section>
+                <section>
+                    <div class="story">
+                        <iframe src="https://www.nytimes.com/interactive/2019/admin/100000002772592.embedded.html?" frameborder="0" scrolling="no" width="800px" height="824"></iframe>
```
A couple of things:
- `width` and `height` attributes are by default in pixels so it should be `800` and not `800px` for width
- `frameborder` attribute is obsolete; by default no border, if you would like to add border do so using CSS (remove)
- `scrolling` attribute is obsolete (remove)

mcrd25 reviewed (changes requested):
# Well done! 👍 Almost 💯. Close to 🚀 But first...

**Please Address review comments and I also suggest you make following changes though it is not necessary for me to give you a 👍**
- Please make menu bar and search icon look better, I suggest you use [font awesome](https://fontawesome.com/)
See original: 
![Screen Shot 2019-05-08 at 8 58 34 AM](https://user-images.githubusercontent.com/31193823/57378405-b8136500-7172-11e9-96d2-7505bb1e9656.png)

- Make buttons look like links, i.e add `cursor: pointer` effects so that they act like links (like original) as opposed to simply buttons. Also maybe add the hover effects if you got the time
- My final issue is that the theory of inflation section is a bit too wide, it should be the same width as the main content of the article. If you may address this it'd make your almost 💯 clone look even better.
See yours: 
![Screen Shot 2019-05-08 at 8 59 48 AM](https://user-images.githubusercontent.com/31193823/57378622-29531800-7173-11e9-9445-52591914e923.png)
vs original :
![Screen Shot 2019-05-08 at 9 26 02 AM](https://user-images.githubusercontent.com/31193823/57378726-5ef80100-7173-11e9-8551-d289f12cbc64.png)

***If you have any questions, let me know here or on slack and I will be glad to clarify with you.
Project really. looks great! Make the required changes, and maybe some of my suggested changes and after a re-review should be able to get the 👍  and merge to master.***

**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

keshav-c committed:
Make changes as recommended in pull request

Use icons from fontawesome.com. Make buttons clickable using
cursor property. Move iframe attributes to css.

keshav-c commented:
Thanks for the review and inputs. I've implemented all changes as you recommended. Please have a look.

keshav-c commented:
Thanks for your review and inputs. I've implemented changes as suggested. Please have a look.

mcrd25 reviewed (approved):
# Wow Guys! The project looks 💯! Well done 🥇! You may now merge to master 👍 
## 🚀 
**Please remember that you can always work to improve in the future of this project. One area you can work on in the future is the nav and its styling! That being said, clone looks really good!**

***If you have any questions, let me know here or on slack and I will be glad to clarify with you.***

**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)
